src/bundle.c

```diff
@@ -721,6 +721,22 @@ static int install_bundles(struct list *bundles, struct list **subs, int current
 	fprintf(stderr, "Installing bundle(s) files...\n");
 	unsigned int list_length = list_len(to_install_files);
 	unsigned int complete = 0;
+	char *fullname;
+	iter = list_head(to_install_files);
+	while (iter) {
+		file = iter->data;
+		iter = iter->next;
+
+		string_or_die(&fullname, "%s/staged/%s", state_dir, file->hash);
+
+		ret = verify_file(file, fullname);
+		if (!ret) {
+			free(fullname);
+			fprintf(stderr, "FAILED HASH CHECK %s\n", fullname);
```

Comment:
I would prefer this message to be more consistent with other errors (i.e. "Error: hash check failed for %s\n")